src/lib/util.js

```diff
@@ -111,6 +116,63 @@ function createDownloadIframe() {
     return iframe;
 }
 
+/**
+ * Checks if the url is a download host, but not the default download host.
+ *
+ * @public
+ * @param {string} url - The URL to check
+ * @return {boolean} - HTTP response
+ */
+export function isNonDefaultDownloadHost(url) {
```

Comment:
Could we use something like `isCustomDefaultDownloadHost`? Feels weird to have isNon.... in the method name